lib/gooddata/lcm/lcm2.rb

```diff
@@ -305,6 +305,12 @@ def perform(mode, params = {})
           skip_actions.include?(action.name.split('::').last)
         end
 
+        sync_mode = params.fetch(:sync_mode, nil)
+        if mode == 'users' && %w[add_to_organization remove_from_organization].include?(sync_mode)
```

Comment:
how about user_filter? does it have these modes and do we need to exclude as well?